san/sanbase_graphql.py

```diff
@@ -193,7 +193,10 @@
             'datetime',
             ('topWords', ['score', 'word'])
         ]
-    }     
+    },
+    'social_projects': {
+        'return_fields':[]
```

Comment:
what is social_projects and why it has return fields `[]` and why in this PR ?

Reply:
is it equivalent to `socialVolumeProjects` from api ?

Reply:
Because that's the only metric in sanbase_graphql.py, which doesn't have an instance in QUERY_MAPPINGS, because it never uses the transform_query_args function. Also, we do need ALL of the functions in sanpy, that's why I added the function in QUERY_MAPPINGS.

Reply:
Maybe I should remove the 'return_fields', I had just added that for consistency

Reply:
do we have an example for this in the README ? I see `social_volume_projects` there

Reply:
```
Because that's the only metric in sanbase_graphql.py, which doesn't have an instance in QUERY_MAPPINGS, because it never uses the transform_query_args function
```
then let's make it use it so it's not handled in this weird inconsitent way